scvi_in_R.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>let's add counts to the getassaydata call https://www.rdocumentation.org/packages/Seurat/versions/3.1.4/topics/GetAssayData</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>